### PR TITLE
Fix #11749: don't warn for hidden files.

### LINKED ISSUE #11749: Build of `doc-stdlib` produces dozens of warnings
These warnings say `Warning: foo will be hidden from the index`

My guess is that this should not be a warning, but I'm not sure. If that's the case, we should stop emitting them. Otherwise, they should be addressed.

@coq/doc-maintainers

### PR DESCRIPTION
**Pull request by Zimmi48**
Fixes / closes #11749

@maximedenes Should we remove the "echo" entirely?

### CODE REVIEW

**Comment by maximedenes** [comment]
I don't get what this output carries, since it reflects directly what was specified manually. I'd remove it altogether.

**Comment by Zimmi48** [comment]
Indeed, I don't believe it brings much either.

File: doc/stdlib/make-library-index [hunk 1]
```diff
@@ -36,7 +36,8 @@ for k in $LIBDIRS; do
                 fi
 	    else
 	        if [ $h = 0 ]; then
-		    echo Warning: $k/$b.v will be hidden from the index
+		    # Skipping file from the index
+                    :
```
Comment (maximedenes) [inline doc/stdlib/make-library-index]: What's this `:` for?

Why not just invert the condition?

Comment (Zimmi48) [reply doc/stdlib/make-library-index]: `:` is the bash operator for a no-op

It seemed to me that the logic was clearer that way: explicitly skip the file from the index.

**Comment by Zimmi48** [comment]
@maximedenes Do you want to assign this one? Let me know if the answer to your comment was satisfying or if you request some changes.

**Review by maximedenes — APPROVED** [review]

**Comment by maximedenes** [comment]
I assume the perennial failure is unrelated.

**Pull request merged**